src/libraries/Microsoft.Extensions.Primitives/src/StringSegment.cs

```diff
@@ -193,7 +193,7 @@ public static int Compare(StringSegment a, StringSegment b, StringComparison com
         /// Indicates whether the current object is equal to another object of the same type.
         /// </summary>
         /// <param name="obj">An object to compare with this object.</param>
-        /// <returns><see langword="true" /> if the current object is equal to the other parameter; otherwise, <see langword="false" />.</returns>
+        /// <returns><see langword="true" /> if the current object is equal to <paramref name="obj">; otherwise, <see langword="false" />.</returns>
```

Comment:
This broke the build 🫠:

```suggestion
        /// <returns><see langword="true" /> if the current object is equal to <paramref name="obj"/>; otherwise, <see langword="false" />.</returns>
```